src/campaign.cairo

```diff
@@ -144,15 +143,7 @@ mod TokengiverCampaign {
             self.campaign.write(campaign_address, new_campaign);
             self.campaigns.write(count, campaign_address);
             self.count.write(count);
-            self
-                .emit(
-                    CreateCampaign {
-                        owner: caller,
-                        campaign_address,
-                        token_id,
-                        token_giver_nft_address: nft_address
-                    }
-                );
+            self.emit(CreateCampaign { owner: caller, campaign_address, token_id, });
```

Comment:
no, please bring back the field remove here

                     `   token_giver_nft_address: nft_address`